pkg/helm/helm.go

```diff
@@ -170,6 +178,13 @@ func pullCommand(chart string, repo *image.HelmRepository, version, destDir stri
 		args = append(args, "--destination", destDir)
 	}
 
+	if repo.SkipTLSVerify {
+		args = append(args, "--insecure-skip-tls-verify")
+	}
+	if repo.PlainHTTP {
+		args = append(args, "--plain-http")
+	}
```

Comment:
Aren't these two mutually exclusive? `else if` should be more appropriate to convey it.